static/js/modals.js

```diff
@@ -24,13 +24,13 @@
 
```

Comment:
The only changes to this file were implemented by the js linter, no need to review